src/Translator.php

```diff
@@ -143,6 +143,19 @@ private function setModulePaths(): void
                 if (!file_exists("$modulePath/.tx/config")) {
                     continue;
                 }
+                // See if the current "branch" is actually a tag that's been checked out,
+                // rather than a minor or next minor branch.
+                // If so, try to git checkout the what the minor branch for the tag should be.
+                // Read all branches rather than just using `git rev-parse --abbrev-ref HEAD` so that
+                // tags aren't just reported as "HEAD", instead they are "(HEAD detached at 3.0.2)"
+                $branches = $this->exec('git branch', $modulePath);
+                // the current branch will be prefixed with a *, extract what the branch is with a regex
+                preg_match("#(^|\n)\* (.+?)(\n|$)#", $branches, $matches);
```

Comment:
```suggestion
                preg_match("#(^| |\n)\* (.+?)(\n| |$)#", $branches, $matches);
```
My shell has a fancy output if I run `git branch`, which with this will end up reading as a space instead of a new line.

Reply:
Does it still do that if you run it via $this->exec() which uses symfony/process which is supposed to standardise all this sort of stuff?

Reply:
Good shout, I assumed it would just run it with the shell I use in my terminal but it turns out it does use new lines when run via symfony process. How delightfully unexpected.